vnext/Mso.UnitTests/Mso.UnitTests.vcxproj

```diff
@@ -73,6 +71,7 @@
       <AdditionalOptions>/await %(AdditionalOptions) /bigobj</AdditionalOptions>
       <ConformanceMode>true</ConformanceMode>
       <LanguageStandard>stdcpp17</LanguageStandard>
+      <CallingConvention>Cdecl</CallingConvention>
     </ClCompile>
```

Comment:
why?

Reply:
The unit test framework requires Cdecl. On x86, we default to StdCall which won't build.

Reply:
I'll remove LanguageStandard as it's inherited, but until #4096, we still need this here so x86 builds.